src/main/java/frc/robot/Constants.java

```diff
@@ -162,26 +97,63 @@ public static class constDrivetrain {
      */
     public static final double MEASUREMENT_STD_DEV_HEADING = Units.Radians.convertFrom(5, Units.Degrees);
 
+    // -- CONFIGS --
+    public static TalonFXConfiguration DRIVE_CONFIG = new TalonFXConfiguration();
+    public static TalonFXConfiguration STEER_CONFIG = new TalonFXConfiguration();
+    public static CANcoderConfiguration CANCODER_CONFIG = new CANcoderConfiguration();
+
+    // This config is kept separate as it's also used in the MODULE_CONFIG :p
+    public static final Current DRIVE_CURRENT_LIMIT = Units.Amps.of(99999);
+
+    public static final double MIN_STEER_PERCENT = 0.01;
+
+    // Rotational speed (degrees per second) while manually driving
+    public static final AngularVelocity TURN_SPEED = Units.DegreesPerSecond.of(360);
+
+    // -- Motor Configurations --
+    static {
+      // This PID is implemented on each module, not the Drivetrain subsystem.
+      DRIVE_CONFIG.Slot0.kP = 0.18;
+      DRIVE_CONFIG.Slot0.kI = 0.0;
+      DRIVE_CONFIG.Slot0.kD = 0.0;
+      DRIVE_CONFIG.Slot0.kS = 0.0;
+      DRIVE_CONFIG.Slot0.kA = 0.0;
+      DRIVE_CONFIG.Slot0.kV = (1 / REAL_DRIVE_SPEED.in(Units.MetersPerSecond));
+
+      DRIVE_CONFIG.MotorOutput.Inverted = InvertedValue.CounterClockwise_Positive;
+      DRIVE_CONFIG.MotorOutput.NeutralMode = NeutralModeValue.Brake;
+      DRIVE_CONFIG.Feedback.SensorToMechanismRatio = SWERVE_CONSTANTS.driveGearRatio;
+      DRIVE_CONFIG.CurrentLimits.SupplyCurrentLimitEnable = true;
+      DRIVE_CONFIG.CurrentLimits.SupplyCurrentLimit = DRIVE_CURRENT_LIMIT.in(Units.Amps);
+
+      STEER_CONFIG.Slot0.kP = 100;
+      STEER_CONFIG.Slot0.kI = 0.0;
+      STEER_CONFIG.Slot0.kD = 0.14414076246334312;
+
+      STEER_CONFIG.MotorOutput.Inverted = InvertedValue.Clockwise_Positive;
+      STEER_CONFIG.MotorOutput.NeutralMode = NeutralModeValue.Coast;
+      STEER_CONFIG.Feedback.SensorToMechanismRatio = SWERVE_CONSTANTS.steerGearRatio;
+      STEER_CONFIG.ClosedLoopGeneral.ContinuousWrap = true;
+
+      CANCODER_CONFIG.MagnetSensor.SensorDirection = SensorDirectionValue.CounterClockwise_Positive;
+    }
+
     public static class AUTO {
       // This PID is implemented on the Drivetrain subsystem
-      public static final double AUTO_DRIVE_P = 8;
-      public static final double AUTO_DRIVE_I = 0;
-      public static final double AUTO_DRIVE_D = 0;
+      public static final PIDConstants AUTO_DRIVE_PID = new PIDConstants(8, 0.0, 0.0);
 
-      public static final double AUTO_STEER_P = 2.5;
-      public static final double AUTO_STEER_I = 0.0;
-      public static final double AUTO_STEER_D = 0.0;
+      public static final PIDConstants AUTO_STEER_PID = new PIDConstants(2.5, 0.0, 0.0);
 
```

Comment:
Might be good to set these to match our comp bot as they worked pretty well with 125 pounds
```suggestion
      public static final PIDConstants AUTO_DRIVE_PID = new PIDConstants(9, 0.0, 0.0);

      public static final PIDConstants AUTO_STEER_PID = new PIDConstants(5.6, 0.0, 0.0);

```